internal/inputs/inputs.go

```diff
@@ -0,0 +1,7 @@
+package inputs
+
+import "github.com/sethvargo/go-githubactions"
+
+func GetBool(name string) bool {
+	return githubactions.GetInput(name) == "true"
```

Comment:
```suggestion
	return strings.EqualFold(githubactions.GetInput(name), "true")
```

You'll also have to import `strings`

https://pkg.go.dev/strings#EqualFold